### PR TITLE
Update package versions

### PR DESCRIPTION
**Pull request by phil-allen-msft**
Regular update of package versions

### CODE REVIEW

File: src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj [hunk 1]
```diff
@@ -16,6 +16,8 @@
     <PackageReference Include="Microsoft.Build.Framework" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.IO.Redist" Condition="'$(TargetFramework)' == '$(NetFxTFM)'" />
```
Comment (jimmylewis) [inline src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj:19]: Are these being added as a means to pin the new versions?  Could we instead set CentralPackageTransitivePinningEnabled in Directory.Packages.props so that transitive references will respect the version specified there, but we're not adding new primary references?  (For example, if the dependency tree changes, adding this as a primary package means we'll always reference it even if it's no longer used.)

Comment (phil-allen-msft) [reply src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj:19]: (Trying it out)

Comment (phil-allen-msft) [reply src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj]: Double checked and all is well.

File: src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj [hunk 2]
```diff
@@ -184,7 +184,7 @@
     <PackageReference Include="Microsoft.WebTools.Languages.Json.VS" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Nuget.VisualStudio" />
-  </ItemGroup>
+   </ItemGroup>
```
Comment (jimmylewis) [inline src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj:187]: nit: unnecessary change in this file (and bad indent)

**Review by jimmylewis — APPROVED** [review]

File: src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj (hunk 2 above)
Comment (phil-allen-msft) [reply src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj]: Correct and corrected.
